logfire/_internal/main.py

```diff
@@ -739,7 +756,7 @@ def with_settings(
             tags=self._tags + tuple(tags),
             sample_rate=self._sample_rate,
             console_log=self._console_log if console_log is None else console_log,
-            otel_scope=self._otel_scope if custom_scope_suffix is None else f'logfire.{custom_scope_suffix}',
+            otel_scope=(self._otel_scope if custom_scope_suffix is None else f'logfire.{custom_scope_suffix}'),
```

Comment:
I'm seeing a lot of formatting changes like this that are unrelated to the PR. Please check that you're using `ruff==0.5.4` for formatting and revert these changes.